src/internal/observable/from.ts

```diff
@@ -4,7 +4,7 @@ import { ObservableInput, SchedulerLike, ObservedValueOf } from '../types';
 import { scheduled } from '../scheduled/scheduled';
 
 export function from<O extends ObservableInput<any>>(input: O): Observable<ObservedValueOf<O>>;
-/** @deprecated use {@link scheduled} instead. */
+/** @deprecated The scheduler argument is deprecated, use scheduled. Details: https://rxjs.dev/deprecations/scheduler-argument */
```

Comment:
Do backticks not work here?

Reply:
I think my concern is this reads like we're saying "Don't use a scheduler, use scheduled" and to people who don't yet know about `scheduled` it might read like "don't schedule, schedule!" haha.

Reply:
backticks and general md support is IDE specific, we could just rephrase it to 'use the scheduled operator' and I think this way it's pretty clear